cluster/gce/container-linux/configure-helper.sh

```diff
@@ -516,7 +516,9 @@ function start-kubelet {
     if [[ "${REGISTER_MASTER_KUBELET:-false}" == "true" ]]; then
       flags+=" --api-servers=https://${KUBELET_APISERVER}"
       flags+=" --register-schedulable=false"
-      flags+=" --register-with-taints=node.alpha.kubernetes.io/ismaster=:NoSchedule"
+      if [[ "${ENABLE_MASTER_NOSCHEDULE_TAINT:-false}" == "true" ]]; then
+        flags+=" --register-with-taints=node.alpha.kubernetes.io/ismaster=:NoSchedule"
```

Comment:
> You reviewed and lgtm'd this taint key

I guess I was thinking this was an experimental feature because we still have Unschedulable. Maybe.